Services/Concrete/ThirdParties/ThirdPartiesServiceFactory.cs

```diff
@@ -22,6 +22,11 @@ public static class ThirdPartiesServiceFactory
 			new ThirdParty
 			{
 				Name = "csgo-stats",
```

Comment:
This should be renamed to `csgo-stats.net` to avoid confusion between both, it's used by the UI.

Reply:
Actually, it looks like the `Url` property is what is displayed in the UI. But I've renamed it for consistency.

Reply:
It's used in the confirmation dialog, thanks